operations.rb

```diff
@@ -0,0 +1,37 @@
+require './store'
+class Operations
+  def initialize
+    @store = Store.new
+  end
+
+  def store_books(title, author)
+    books = []
+    book = { title: title, author: author }
+    books = @store.read_file('books.json') if File.exist?('books.json')
+    books.push(book)
+    @store.write_file('books.json', books)
+  end
+
+  # rubocop:disable Style/OptionalBooleanParameter
+  def store_person(num, age, name, permission = true, specialization = nil)
+    # rubocop:enable Style/OptionalBooleanParameter
+
+    persons = []
+    persons = @store.read_file('persons.json') if File.exist?('persons.json')
+    person = if num == 1
+               { options: num, age: age, name: name, parent_permisson: permission }
+             else
+               { options: num, age: age, name: name, specialization: specialization }
+             end
+    persons.push(person)
+    @store.write_file('persons.json', persons)
+  end
```

Comment:
- [ ] As required for the project, kindly ensure that your application matches the expected requirements. Right now, user isn't able to even create a `Person`(Student or Teacher). Your application raises the error bellow **and as result it is impossible to check if you application can preserve data**
![Screenshot from 2023-07-05 14-45-31](https://github.com/bilalrajput09/OOP-school-library/assets/83042742/44328d58-9454-4b63-96c8-70dfe50e4ff8)
Ensure that you have fixed the above error and **Your application should be able to preserve data once fixed the error.**

Reply:
Fixed
![image](https://github.com/bilalrajput09/OOP-school-library/assets/116716918/ab735902-9a06-4ef9-ab55-30a11187fb2a)